resources/views/contact.blade.php

```diff
@@ -1,5 +1,213 @@
+            <h4 class="text-3xl sm:text-4xl lg:text-5xl text-center xl:text-left font-semibold text-black "><span
+                    class="text-website-normal">{{ __('content.contact.title') }}</span> {{ __('content.contact.title2') }}</h4>
+            <p class="text-md xl:text-lg leading-8 text-black text-center xl:text-left py-6">{{ __('content.contact.subtitle') }}
+            </p>
+            <div class="flex flex-col lg:flex-row">
+                <form action="#" method="POST" class="lg:flex-auto">
+                    <div class="grid grid-cols-1 gap-y-6">
+                        <div>
+                            <label for="e-mail"
+                                class="block text-xs font-semibold leading-6 text-gray-700">{{ __('content.contact.form.email') }}</label>
+                            <div class="mt-2.5">
+                                <input type="text" name="e-mail" id="e-mail" autocomplete="given-e-mail"
+                                    class="block w-full xl:w-[90%]  h-12 rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                            </div>
+                        </div>
+                        <div>
+                            <label for="topic"
+                                class="block text-xs font-semibold leading-6 text-gray-700">{{ __('content.contact.form.title') }}</label>
+                            <div class="mt-2.5">
+                                <input type="text" name="topic" id="topic" autocomplete="topic"
+                                    class="block w-full xl:w-[90%]  h-12 rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
+                            </div>
+                        </div>
+                        <div>
+                            <label for="message"
+                                class="block text-xs font-semibold leading-6 text-gray-700">{{ __('content.contact.form.message') }}</label>
+                            <div class="mt-2.5">
+                                <textarea id="message" name="message" rows="4"
+                                    class="block w-full xl:w-[90%] rounded-md border-0 px-3.5 py-2 min-h-12 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"></textarea>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="flex items-center mt-2 xl:w-[90%]">
+                        <input type="checkbox" name="policyBox" id="policyBox" class="w-3 h-3">
+                        <p class="text-xs font-normal text-gray-600 ml-2 pt-4">Akceptuję regulamin oraz <a href=""
```

Comment:
Would be nice to translate this whole line.

Reply:
Moved the text to content.php, please check if it's split correctly.